.github/ISSUE_TEMPLATE/3_Support_question.md

```diff
@@ -1,6 +1,6 @@
 ---
-name: Support Question
-about: Questions about using Symfony and its components
+name: ⛔ Support Question
+about: See https://symfony.com/support for questions about using Symfony and its components.
```

Comment:
If you add a dot here, you should add it on other `about` section as well... or just remove the ones you added.

Reply:
dots added

Reply:
TBH, I would have removed the ones you added

Reply:
removed :)